### ISSUE TITLE
button.css has an impact on the style of other buttons

### ISSUE THREAD

**Issue by sylvoslee** [np-8/dash-uploader#43]

**Comment by cainmagi** [comment]
Also for `progressbar.css`. The stylesheets from `dash-uploader` will override the stylesheets used by `dash-bootstrap-components`.

**Comment by np-8** [comment]
Yeah thats right. On the other hand it should be so that there are good defaults, but on the other hand adding uploader component to a page should no affect the rest of the page. It would be nice if the dash-uploader component could use the styling used elsewhere on the same site if available. What would be the most appropriate way to handle this?
